tools/rerun/rerun_st.py

```diff
@@ -0,0 +1,146 @@
+import multiprocessing
+from openpilot.tools.lib.logreader import LogReader
+from openpilot.tools.lib.route import Route
+import subprocess
+import sys
+import os
+import capnp
+
+
+NUM_CPUS = 1 # Number of CPUS to use in pool
```

Comment:
our styling is 2 spaces for inline comments. but this seems very self explanatory

```suggestion
NUM_CPUS = 1
```